### ISSUE TITLE
Mainspec/Offspec button naming

### ISSUE THREAD

**Issue by aweinmann** [Road-block/BastionEPGP#7]
Hi,
Would it be possible to have the button names set in the settings menu?
And maybe additional repsonse buttons (with different price %)?

Thanks a lot

**Comment by Road-block** [comment]
Additional response buttons and price categories is a fairly substantial rewrite (the library used for static popups has a max of 3, so I'd have to write my own from scratch). I'd say unlikely at this point. 

Response button names / chatlinks being customizable is somewhat simpler on the local level but how would that work across a guild?  
You'd have to tell everyone to go into their addon options and type something in, those are not shared options.  
I'm hesitant to make them shared as well, I try to limit addon comms to the absolute minimum necessary, there's enough addons (and some that really need the bandwidth for real-time combat information like threat) competing for bandwidth as it is.  

It's also designed to be optional for simple members. I can't think of a way to do what you want while keeping the basic design of the addon intact, but I'll keep the ticket open for a bit and have a think.

**Comment by aweinmann** [comment]
Thanks for the feedback, CEPGP (https://github.com/Alumian/CEPGP-Retail) has some guildrank based settings enforce system. dont know how they handle the communication but just as a hint.

Anyway thanks for the great addon!
is there any roadmap on future features you would like to add?

**Comment by Road-block** [comment]
Yeah I'm not one to critique other addons but I'm never doing [this](https://github.com/Alumian/CEPGP-Retail/blob/master/Communications.lua)  
I consider BastionEPGP mostly feature complete.  

The items on my TODO list for v2.x are

- Let guild leader import full standings from a backup/export.
- Import reserves (for the plusroll mode) from discord (if I can find a graceful way to match names to items)
- Built-in pricelist needs a bit of maintenance. It has some horde bias as well as private server bias from when it was originally mapped out. Now due to the way EPGP works even an imperfect pricelist is not a huge issue but I would like item prices to more closely match item desirability and possibly offer different prices for alliance vs horde.

Last one is more of a long term goal as BiS lists are not set in stone and as said even an imperfect price list is in 99% of cases "good enough" for spreading loot around a guild fairly.
